app/controllers/admin/event_schedules_controller.rb

```diff
@@ -31,5 +31,11 @@ def destroy
     def event_schedule_params
       params.require(:event_schedule).permit(:schedule_id, :event_id, :room_id, :start_time)
     end
+
+    def parse_errors(event_schedule)
```

Comment:
Why not? When clicking save I am scheduling several events, so I need to link the error messages to render all together and I use that function to allow that this is done. I add the name of the event (taking into account that validation may have failed), the the errors in brackets and a dot at the end. And I have to do that in `create`, `update` and `destroy`

Look:

![image](https://cloud.githubusercontent.com/assets/16052290/17629034/9feb678e-60b8-11e6-880c-b5d70ac2f35b.png)